src/components/ebay-icon/index.js

```diff
@@ -26,6 +26,8 @@ function init() {
 }
 
 function getTemplateData(state, input, out) {
+    const custom = input.custom || 'false';
+    const isCustom = custom === 'true';
```

Comment:
Looks like `custom` is a boolean attribute. As per our [guidelines](https://medium.com/@senthil_hi/building-a-ui-component-in-2017-and-beyond-1f6d5c4d464#f7e3), if an attribute is boolean, just the presence of the attribute on an element represents the true value. Can you please update it accordingly? This should also be reflected in docs.